test/sqlite/test.out

```diff
@@ -1,7 +1,5 @@
 count(*) = 2
 
-datetime('2012-04-16 12:35:57', '+1 days') = 2012-04-17 12:35:57
```

Comment:
I can't figure out what change led to this no longer being needed?

Reply:
I'm running a completely different test file, so this is a brand new test output.   The fact that github thinks I renamed and modified and existing file is misleading.

Reply:
Ah, makes sense.